modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -2614,9 +2614,7 @@ void ONNXImporter::parseShape(LayerParams& layerParams, const opencv_onnx::NodeP
     if (isDynamicShape)
     {
         CV_LOG_ERROR(NULL, "DNN/ONNX(Shape): dynamic 'zero' shapes are not supported, input " << toString(inpShape, node_proto.input(0)));
-        // FIXIT repair assertion
-        // Disabled to pass face detector tests from #20422
-        // CV_Assert(!isDynamicShape);  // not supported
+        CV_Assert(!isDynamicShape);  // not supported
```

Comment:
Updated detection model passes this assertion.
But recognition model doesn't.

Reply:
Problem fixed. It was using before-update version of YuNet in the regression test of face recognition.